ci-operator/step-registry/baremetalds/two-node/fencing/extended/baremetalds-two-node-fencing-extended-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: baremetalds-two-node-fencing-extended
+  steps:
+    allow_skip_on_success: true
+    allow_best_effort_post_steps: true
+    cluster_profile: equinix-edge-enablement
+    pre:
+      - chain: baremetalds-ipi-pre
+    test:
+      - ref: openshift-extended-test
+    post:
+      - chain: baremetalds-ipi-post
+    env:
+      CLUSTERTYPE: baremetal
```

Comment:
Change this line, since we're using vms in the metal machine you can remove this line and switch it over to `PACKET_OS: rocky_9` which will resolve the pip problem.